packages/plans-grid-next/src/components/actions.tsx

```diff
@@ -151,7 +114,6 @@ const LoggedInPlansFeatureActionButton = ( {
 	}
 
 	if ( current && planSlug !== PLAN_P2_FREE ) {
```

Comment:
In reference to my other comment (https://github.com/Automattic/wp-calypso/pull/90611/files#r1597666873) can we drop the check here too? 🤔 

```suggestion
	if ( current ) {
```

Reply:
[07bdebc](https://github.com/Automattic/wp-calypso/pull/90611/commits/07bdebc7148a8640d762f3c1f5df01448a5fbda3)